### PR TITLE
Process regarding the generic dataset for testing on DEV and TEST

### PR DESCRIPTION
**Pull request by djorgensendk**
Creating of a generic dataset for tests to run on the ENDK's Datahub DEV and TEST environments is ongoing.
The dataset will be shared in the Green Energy Hub community. This document introduces the test dataset, and outlines the process of how it in the future can be updated when needed

### CODE REVIEW

**Review by ASQ-EN — APPROVED** [review]
Comment added to EIC clarification
data document is pending

File: docs/test-data/process-regarding-test-data.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Test data in Green Energy Hub
+
 The ENDK Datahub business will generate and own a generic dataset containing the master data as listed below. This dataset is intented to be the foundation for testing activities of Green Energy Hub on the DEV and TEST environments.
+
 * Organisation name and IDs (GLN and EIC) for a number of:
```
[the remark this answers is not in the document]
Comment (djorgensendk) [reply docs/test-data/process-regarding-test-data.md]: A footnote explaining this has been added

File: docs/test-data/process-regarding-test-data.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# Test data in Green Energy Hub
+
+The ENDK Datahub business will generate and own a generic dataset containing the master data as listed below. This dataset is intented to be the foundation for testing activities of Green Energy Hub on the DEV and TEST environments.
+
+* Organisation name and IDs (GLN and EIC - see Footnote 1) for a number of:
+    * Energy Suppliers
+    * Balance Responsible parties
+    * Grid operators
```
Comment (prtandrup) [inline docs/test-data/process-regarding-test-data.md:8]: nit: Please use:
```suggestion
    * Grid Access Providers
```

File: docs/test-data/process-regarding-test-data.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+The dataset's IDs will be unique to avoid problems when the test data migration from the ENDK Datahub 2 commence.
+
+As evident from the master data attributes listed above, this dataset will only contain a limited set of data. However, using the dataset as the foundation, teams will be able to create mocked data and associate these with a meaningful set of master data. Generation of mock data (and the ownership of these) will be the responsibility of the team that needs to run a test suite, e.g. intra domain integration tests.
+
+## Where to find and how to update
+
+The generic dataset will be made accessible as an Excel spreadsheet in the green-energy-hub repository in the [dataset](./dataset/) folder.
+
+Thereby the generic dataset will be available to the GEH community. As Green Energy Hub grows and matures, this dataset as expected to do so as well. But as of now, June 2021, the dataset is scoped to the aforementioned master data.
```
Comment (prtandrup) [inline docs/test-data/process-regarding-test-data.md:22]:
```suggestion
Thereby the generic dataset will be available to the GEH community. As Green Energy Hub grows and matures, this dataset is expected to do so as well. But as of now, June 2021, the dataset is scoped to the aforementioned master data.
```

File: docs/test-data/process-regarding-test-data.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+However, if additional generic attributes needs be added to the dataset, please send this request to team Mighty Ducks by raising an issue in the green-energy-hub repository and assign it to: ASQ-EN, djorgensendk, and Renetnielsen. For the community to be able to work with a consistent dataset, updates to it needs to be coordinated.
+
+Footnote 1: The EIC field will be available, but it value is not yet mandatory, as the format is yet to be determined.
```
Comment (prtandrup) [inline docs/test-data/process-regarding-test-data.md:26]:
```suggestion
Footnote 1: The EIC field will be available, but its value is not yet mandatory, as the format is yet to be determined.
```

**Review by prtandrup — APPROVED** [review]
LGTM with a few nits

**Pull request merged**